README.md

```diff
@@ -37,6 +37,24 @@ Then, in the browser tab that opens, select the notebook you would like to run.
 Alternatively, to run the tutorial as a script, skip the Jupyter launch command and run the tutorial's `.py` file directly (e.g. `python spam_tutorial.py`).
 
 
+## Running with Docker
+We've included a Docker setup for our tutorials to make setup easy.
+First, make sure you have [Docker installed](https://docs.docker.com/install/) on your machine.
+To build and run a Docker image for a tutorial, use `scripts/launch_docker.py` with the `--build` flag.
+For example, run the following for the `spam` tutorial:
+
+```bash
+python3 scripts/launch_docker.py spam --build
+```
+
+This will make a Jupyter notebook server available on port 8888.
+You can change the port with the `--port` option.
```

Comment:
Even if I change the port, the corresponding localhost address for the jupyter notebook always uses 8888 (in the link shown in the output).
To access the notebook, open this file in a browser:
        file:///root/.local/share/jupyter/runtime/nbserver-6-open.html
    Or copy and paste one of these URLs:
        http://d3761a94713c:8888/?token=***********************************************************
     or http://127.0.0.1:8888/?token=***********************************************************

Reply:
That's what's running inside the Docker container (will always be 8888). Port 8888 should get forwarded to (and be accessible at) your custom port on the host (your machine). lmk if it's not working for you

Reply:
Fixed!